core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -183,11 +183,16 @@ public BlockInStream getInStream(long blockId, InStreamOptions options,
         || options.getStatus().getPersistenceState().equals("TO_BE_PERSISTED")) {
       blockWorkerInfo = getEligibleWorkers();
       workerPool = blockWorkerInfo.stream().map(BlockWorkerInfo::getNetAddress).collect(toSet());
+      if (workerPool.isEmpty()) {
+        throw new UnavailableException(
+            "No Alluxio worker available. Check that your workers are still running");
```

Comment:
Consider making this a constant since we test it?

Reply:
prefer to keep it inlined to improve readability here and in the test